executors.go

```diff
@@ -84,6 +159,26 @@ func (c *Connection) ValidateAndCreate(model interface{}, excludeColumns ...stri
 // Create add a new given entry to the database, excluding the given columns.
 // It updates `created_at` and `updated_at` columns automatically.
 func (c *Connection) Create(model interface{}, excludeColumns ...string) error {
+	v := reflect.Indirect(reflect.ValueOf(model))
+	if v.Kind() == reflect.Slice || v.Kind() == reflect.Array {
+		var err error
+		for i := 0; i < v.Len(); i++ {
+			val := v.Index(i)
+			if val.Kind() != reflect.Ptr {
+				val = val.Addr()
+			}
+			err = c.Create(val.Interface(), excludeColumns...)
+			if err != nil {
+				return err
+			}
+		}
+		return err
+	}
+
+	if c.eager {
```

Comment:
would this statement be better at the beginning instead of after all the reflection work?